Assets/Shopify/UIToolkit/Theme Controllers/Editor/SingleProductThemeControllerEditor.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Shopify.UIToolkit.Editor {
+    using UnityEngine;
+    using UnityEditor;
+    using System;
+
+    [CustomEditor(typeof(SingleProductThemeController))]
+    public class SingleProductThemeControllerEditor : Editor {
+        public SingleProductThemeController Target {
+            get {
+                return target as SingleProductThemeController;
+            }
+        }
+
+        public void OnEnable() {
+            BindThemeIfPresent();
+        }
+
+        public override void OnInspectorGUI() {
+            if (Target.Theme == null) {
```

Comment:
These things are going to be hard to test... I might create an EditorView layer that is testable and just have the editor call out to it... Right now what this does is pretty trivial though